lib/katello/plugin.rb

```diff
@@ -512,6 +512,12 @@ def katello_template_setting_values(name)
         description: N_("Default download policy for Smart Proxy syncs (either 'inherit', immediate', or 'on_demand')"),
         collection: proxy_download_policies
 
+      setting 'validate_host_lce_content_source_coherence',
+        type: :boolean,
+        default: true,
+        full_name: N_('Validate host/lifecycle environment/content source coherence'),
+        description: N_("Validate that a host's assigned lifecycle environment is synced by the smart proxy from which the host will get its content")
```

Comment:
```suggestion
        description: N_("Validate that a host's assigned lifecycle environment is synced by the smart proxy from which the host will get its content at registration time")
```

The change content source UI will still block this.

Reply:
There are still some places in the UI that won't, like the legacy Content Host UI and Content host bulk actions. So it's not just registration time.

Reply:
Think it's worth noting that the change content source UI alone won't respect the setting then?